### PR TITLE
Saleor 1737 tests for shipping methods

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created tests for shipping methods

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.
1. [ ] The changes are tested in different browsers (Chrome, Firefox, Safari).
1. [ ] The changes are tested in light and dark mode.

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1737 Tests for Configuration -> Shipping Methods](https://app.clickup.com/t/2549495/SALEOR-1737)

**Review by dominik-zeglen — APPROVED** [review]

File: cypress/integration/configuration/shippingMethod.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      email: "test@example.com",
+      variantsList,
+      address: plAddress,
+      auth: "token"
+    }).then(checkout => {
+      const isShippingAvailable = isShippingAvailableInCheckout(
+        checkout,
+        shippingName
+      );
+      /* eslint-disable no-unused-expressions */
```
Comment (jwm0) [inline cypress/integration/configuration/shippingMethod.js:160]: See this:
https://github.com/cypress-io/eslint-plugin-cypress#chai-and-no-unused-expressions

Comment (karola312) [reply cypress/integration/configuration/shippingMethod.js:160]: ok, I will install plugin and remove eslint comments

**Comment by jwm0** [comment]
It looks like snapshots are not matching correctly which causes jest to fail. Run `npx jest src/storybook/Stories.test.ts -u` to update them

**Review by jwm0 — COMMENTED** [review]
It looks like snapshots are not matching correctly which causes jest to fail. Run:
```
npx jest src/storybook/Stories.test.ts -u
```
to update them

**Pull request merged**